framework/shader_compiler.h

```diff
@@ -33,7 +33,7 @@ namespace vkb
 {
 /// Helper class to generate SPIRV code from GLSL source
```

Comment:
With the compiler now being able to support other languages, the comments need to be changed too. They still refer to GLSL only.

Reply:
That comment still needs to be updated.

Reply:
Updated